### DISCUSSION TITLE
Vote to retire @dumez-k from Stacks Maintainer team

### DISCUSSION THREAD

**Discussion by dumez-k** [paketo-buildpacks/feedback#151] [Stacks Team]
Hey team, with my increased responsibilities on the TBS side, along with @sophiewigmore  and @brayanhenao stepping up, this seems like a good time to shift to an emeritus status and remove myself from the Stacks Maintainers team. Still will always be happy to help however if any context or anything is needed!

@paketo-buildpacks/stacks-maintainers can I get a 👍 / 👎 ?

**Comment by sophiewigmore** [comment]
We'll miss you!!

**Comment by ryanmoran** [comment]
With a super-majority vote to affirm, @dumez-k will be retired and moved to emeritus status.
